segment_data/strava_segment_data_load.py

```diff
@@ -0,0 +1,20 @@
+import requests
+import json
+
+ACCESS_TOKEN = 'ae6026e2583caaf9348db334197e5c9484c751b3'
+REDWOOD_SEGMENTS = [6333339, 799775, 6236074, 653093, 6236074, 3636730]
```

Comment:
You might want to load these from a file too so you can update it without changing the code.  Might make sense to do that later though.